test/Factory/AcceptListenerFactoryTest.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * @license   http://opensource.org/licenses/BSD-3-Clause BSD-3-Clause
+ * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ */
+
+namespace ZFTest\Versioning\Factory;
+
+use Interop\Container\ContainerInterface;
+use PHPUnit_Framework_TestCase as TestCase;
+use ReflectionClass;
+use ZF\Versioning\AcceptListener;
+use ZF\Versioning\Factory\AcceptListenerFactory;
+
+class AcceptListenerFactoryTest extends TestCase
+{
+    public function setUp()
```

Comment:
Neither is required.

- Visibility only needs to be as visible or more visible than the parent class declares.
- PHPUnit declares `setUp()` as a no-op, so overriding methods do not need to call on the parent.